application/views/admin/token/massive_actions/_update.php

```diff
@@ -419,15 +419,12 @@ class="YesNoDatePicker form-control action_datepickerUpdateHiddenField"
 <?php App()->getClientScript()->registerScript("Tokens:MassActionUpdateView_Scripts", "
 
    var bindBSSwitch = function(formGroup){
-       console.log(\"bindBSSwitch run on:\",formGroup);
-    //Script to update the completed settings
-    formGroup.find('.YesNoSwitch').on('switchChange.bootstrapSwitch', function(e, state){
-        
-        formGroup.find('.selector_datechange').css('display', (state ? '' : 'none'));
-        //formGroup.find('.selector_submitField').val(state ? 'Y' : 'N');
-
-    });
-   };
+        console.log(\"bindBSSwitch run on:\",formGroup);
```

Comment:
Do we need this debugging statement?

Reply:
I don't remember if we had a proper logger in JS? That's only on if debug = 2, with different logging levels.

Reply:
Fixed